history/characters/15000_gurubashi.txt

```diff
@@ -1053,7 +1056,7 @@
 	trait=trusting trait=education_intrigue_2
 	499.7.25={ birth=yes trait=creature_troll }
 	515.7.25={
-		# trait=class_rogue_2
+		effect={ set_variable = { name = wc_rogue_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_1_magic_trait_value days = 10 } }
```

Comment:
These values don't exist so a lot of errors appeared.

Reply:
Ported them from the other branch, so it should be fixed now